packages/cozy-stack-client/src/Collection.js

```diff
@@ -40,6 +40,18 @@ export const isIndexConflictError = error => {
   return error.message.match(/error_saving_ddoc/)
 }
 
+/**
+ * Helper to identify a not-used index
+ *
+ * @param {string} warning - The warning returned by CouchDB
+ * @returns {boolean} Whether or not this is a not-used index warning
+ */
+export const isIndexNotUsedWarning = warning => {
+  return warning.match(
+    /was not used because it does not contain a valid index for this query/
+  )
```

Comment:
Are we sure that we have the same message between Couch 2 and Couch 3 ? (sorry for this kind of question)

Reply:
No need to be sorry, those are valid questions ;)
And yes, I checked on Couchdb 2.3 and 3.2